src/app/containers/SignupContainer/SignupContainer.js

```diff
@@ -201,11 +201,7 @@ const SignupContainer = ({ history, onLogin, stopRedirect }) => {
                         )}
 
                         {signupState === SignupState.Payment && (
-                            <PaymentStep
-                                model={model}
-                                paymentAmount={selectedPlan.price.total}
-                                onPay={(...rest) => withCreateLoading(handlePayment(...rest))}
-                            >
+                            <PaymentStep model={model} paymentAmount={selectedPlan.price.total} onPay={handlePayment}>
```

Comment:
`onPay` returns `model, paymentParameters`?

Reply:
yes

Reply:
onPay is not the same one as on `Payment` component. Don't remember what problem exactly this solved, maybe it's not needed to return model there anymore, but it was less fragile this way.